File: SistemaBancario/src/ContaSimples.cpp
#include "ContaSimples.h"

ContaSimples::ContaSimples(int numero, string cliente){
		this->numero = numero;
		this->cliente = cliente;
		this->saldo = 0;
}

ContaSimples::~ContaSimples(){
	//std::cout << "'destruindo' base\n";
}

int ContaSimples::getNumero(){
	return this->numero;
}

double ContaSimples::getSaldo(){
	return this->saldo;
}

void ContaSimples::efetuaDeposito(double valor){
	this->saldo += valor;
}

bool ContaSimples::efetuaSaque(double valor){
	if(this->saldo >= valor){
		this->saldo -= valor;
		return true;
	}
	return false;
}

File: SistemaBancario_java/src/ContaSimples.java

public class ContaSimples {
	private int numero;
	private String cliente;
	private double saldo;

	public ContaSimples(int numero, String cliente){
		this.numero = numero;
		this.cliente = cliente;
		this.saldo = 0;
	}

	public ContaSimples(){
	}

	int getNumero(){
		return this.numero;
	}

	double getSaldo(){
		return this.saldo;
	}
	
	void setSaldo(double valor){
		this.saldo = valor;
	}

	void efetuaDeposito(double valor){
		this.saldo += valor;
	}

	boolean efetuaSaque(double valor){
		if(this.saldo >= valor){
			this.saldo -= valor;
			return true;
		}
		return false;
	}

}
